src/puppetlabs/trapperkeeper/services/webserver/jetty9_core.clj

```diff
@@ -626,12 +626,11 @@
     (.setHandlers hc (into-array Handler [(:handlers webserver-context)]))
     (let [shutdown-timeout (when (not (nil? (:shutdown-timeout-seconds options)))
                              (* 1000 (:shutdown-timeout-seconds options)))
-          maybe-zipped (if (or (not (contains? options :gzip-enable))
-                               (:gzip-enable options))
+          maybe-zipped (if (:gzip-enable options true)
```

Comment:
I could put a `get` in front of it if you think that'd look better but, yeah, it "just works" with this.